README.md

```diff
@@ -100,12 +100,12 @@ To see the daemon options:
 
 Start the daemon with the options to indicate the configuration files path:
 
-`./src/thingd -n -c confs/credentials.conf -d confs/device.conf -r confs/rabbitmq.conf`
+`./src/thingd -n -c confs/credentials.conf -d confs/device.conf -r confs/cloud.conf`
 
 ### How to check for memory leaks and open file descriptors
 
 `valgrind --leak-check=full --track-fds=yes ./src/thingd -n -c `
-`confs/credentials.conf -d confs/device.conf -r confs/rabbitmq.conf`
+`confs/credentials.conf -d confs/device.conf -r confs/cloud.conf`
```

Comment:
@vitbaq @kbcvcbk @luigiluz @CaioCordeiro What do you think about keeping or changing the '-r' flag?

Reply:
I think it's better to change to -C

Reply:
It can be confusing because of -c, idk. What are your suggestions?

Reply:
Since we already use -c for the credentials file, if its possible we can use two letters, something like "-cl" or even "m" as a shortcut for message.

Reply:
I think -cl its the most intuitive, but if we are using 2 letters in 1 flag, shouldn't we use it in the others? Since they have the same purpose.

Reply:
I think we can use "cl" in this case and change the other ones at another time.

Reply:
Agreed

Reply:
idk if we can use 2 letters, what do you think @alla3?

Reply:
We can't use two letters with getopt. :/

Reply:
We can use -r or -C. What do you think guys? @kbcvcbk @alla3 @luigiluz 

-r because of rabbitmq (the cloud used protocol)
-C because of Cloud

Reply:
I agree @vitbaq, both option seen reasonable to me.

Reply:
ok, so let's continue with -r

Reply:
actually, I'd propose we could use -d for device configuration, instead of -c, so we could use -c for cloud configuration. I think that's a better option because it keeps the mnemonics in place.

Reply:
But we already use -d for device conf
-c -> credentials

Reply:
-p